src/gui/optionsdlg.cpp

```diff
@@ -77,13 +77,14 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     setModal(true);
 
     // Icons
-    m_ui->tabSelection->item(TAB_UI)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_APPEARANCE)->setIcon(GuiIconProvider::instance()->getIcon("preferences-desktop"));
+    m_ui->tabSelection->item(TAB_BEHAVIOUR)->setIcon(GuiIconProvider::instance()->getIcon("gear"));
```

Comment:
FDO stands for FreeDesktop.org, and [here](https://standards.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html) is the icon theme specification. You may also look at `QIcon::fromTheme()` documentation.